.github/workflows/compile.yml

```diff
@@ -0,0 +1,77 @@
+name: Build policies
```

Comment:
Is it a typo? I am not sure I follow which 'policies' are referred here...

Reply:
It refers to build policies (on push, on pull request), but I think I lost another `-name` somewhere to name the actual test.